File: OpenIM-SDK/src/main/java/io/openim/android/sdk/listener/OnPutFileListener.java
package io.openim.android.sdk.listener;


public interface OnPutFileListener {
    /**
     * Upload completed - onSuccess
     * @param size file size
     * @param url uploaded URL
     * @param type file type
     */
    default void complete(long size, String url, long type){}

    default void  hashPartComplete(String var1, String var2){}

    default void  hashPartProgress(long var1, long var3, String var5){}

    default void  open(long var1){}

    default void  partSize(long var1, long var3){}

    /**
     * Upload progress
     */
    default void uploadComplete(long size, long progress, long l2){}

    default void  uploadID(String var1){}

    default void  uploadPartComplete(long var1, long var3, String var5){}
}

File: OpenIM-SDK/src/main/java/io/openim/android/sdk/listener/OnPutFileListener.kt
package io.openim.android.sdk.listener

interface OnPutFileListener {
    fun complete(size: Long, url: String, type: Long) {}
    fun hashPartComplete(var1: String, var2: String) {}
    fun hashPartProgress(var1: Long, var3: Long, var5: String) {}
    fun open(var1: Long) {}
    fun partSize(var1: Long, var3: Long) {}
    fun uploadComplete(size: Long, progress: Long, l2: Long) {}
    fun uploadID(var1: String) {}
    fun uploadPartComplete(var1: Long, var3: Long, var5: String) {}
}
